jobs/src/main/scala/dcos/metronome/model/ScheduleSpec.scala

```diff
@@ -12,23 +12,23 @@ import scala.concurrent.duration._
 case class ScheduleSpec(
   id:                String,
   cron:              CronSpec,
-  timeZone:          DateTimeZone      = ScheduleSpec.DefaultTimeZone,
+  timeZone:          ZoneId            = ScheduleSpec.DefaultTimeZone,
   startingDeadline:  Duration          = ScheduleSpec.DefaultStartingDeadline,
   concurrencyPolicy: ConcurrencyPolicy = ScheduleSpec.DefaultConcurrencyPolicy,
   enabled:           Boolean           = ScheduleSpec.DefaultEnabled) {
   def clock: Clock = ScheduleSpec.DefaultClock
 
-  def nextExecution(after: DateTime): DateTime = {
-    val localAfter = after.toDateTime(timeZone)
+  def nextExecution(after: Instant): Instant = {
+    val localAfter = ZonedDateTime.ofInstant(after, timeZone)
```

Comment:
Is it possible to use Instant everywhere?

Reply:
this is actually because of the cron library, that schedules the cron based on the local time of the user, not UTC... But I also wondered about this and I am not sure, maybe @meichstedt or timezone wizard @kensipe would have more context, this is just my guess...

Reply:
metronome is the worst! :)   It is constrained by marathon as a lib... it is constrained by cron-utils.   and they all made different date chooses... and they all made changes over time... 
We are moving to the latest marathon which moved to the new Java Date.   I tried to talk cron-utils to do the same... however they were resistant to provide support for a wider client base.   They internally use the 310 BP which is essential Java 8 Dates:  https://github.com/ThreeTen/threetenbp

The short is it would be great to simplify and use Java 8 Dates as much as possible and have internal conversion utils for working with cron-utils.

This current work was necessary to move to the latest marathon.  I support maintain that as our goal... focus on the minimum necessary to get us as quickly as possible on the latest marathon with a jira to rework. (unless it is viewed as just as easy to do now)